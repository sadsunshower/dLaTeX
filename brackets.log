This is pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2019/dev/Debian) (preloaded format=pdflatex 2019.10.3)  21 DEC 2019 17:32
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**testing/brackets.tex
(./testing/brackets.tex
LaTeX2e <2018-12-01>
(/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2018/09/03 v1.4i Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo
File: size10.clo 2018/09/03 v1.4i Standard LaTeX file (size option)
)
\c@part=\count80
\c@section=\count81
\c@subsection=\count82
\c@subsubsection=\count83
\c@paragraph=\count84
\c@subparagraph=\count85
\c@figure=\count86
\c@table=\count87
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
No file brackets.aux.
\openout1 = `brackets.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <7> on input line 7.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <5> on input line 7.
[1

{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}] (./brackets.aux) ) 
Here is how much of TeX's memory you used:
 202 strings out of 494561
 2198 string characters out of 6177454
 56635 words of memory out of 5000000
 3951 multiletter control sequences out of 15000+600000
 3640 words of font info for 14 fonts, out of 8000000 for 9000
 14 hyphenation exceptions out of 8191
 23i,4n,17p,124b,107s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/sh
are/texlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmex10.pfb></usr/share/te
xlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi10.pfb></usr/share/texlive/
texmf-dist/fonts/type1/public/amsfonts/cm/cmmi7.pfb></usr/share/texlive/texmf-d
ist/fonts/type1/public/amsfonts/cm/cmr10.pfb>
Output written on brackets.pdf (1 page, 37388 bytes).
PDF statistics:
 24 PDF objects out of 1000 (max. 8388607)
 16 compressed objects within 1 object stream
 0 named destinations out of 1000 (max. 500000)
 1 words of extra memory for PDF output out of 10000 (max. 10000000)

