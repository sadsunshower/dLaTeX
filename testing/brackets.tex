\documentclass{article}

\begin{document}

Test: brackets

It should catch these: $(\frac{a}{b})$

\[
    (\frac{a}{b})
\]

\[
(\int^a_b c)
\]

\[(\sum^a_b c)\]

\[
    (\left(\frac{a}{b}\right))
\]

But not these: \(\frac{a}{b}\)

\[
    \left(\frac{a}{b}\right)
\]

\[
\left(\frac{a}{b}\right)
\]

\[\left(\frac{a}{b}\right)\]

\end{document}