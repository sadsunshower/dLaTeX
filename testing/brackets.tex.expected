This is dLaTeX version 0.1
{"messages": [{"severity": "warning", "message": "Did you mean to auto-size these brackets with \\left and \\right?", "line": "It should catch these: $(\\frac{a}{b})$", "num": 6, "marker": [24, 37]}, {"severity": "warning", "message": "Did you mean to auto-size these brackets with \\left and \\right?", "line": "    (\\frac{a}{b})", "num": 9, "marker": [4, 17]}, {"severity": "warning", "message": "Did you mean to auto-size these brackets with \\left and \\right?", "line": "(\\int^a_b c)", "num": 13, "marker": [0, 12]}, {"severity": "warning", "message": "Did you mean to auto-size these brackets with \\left and \\right?", "line": "\\[(\\sum^a_b c)\\]", "num": 16, "marker": [2, 14]}, {"severity": "warning", "message": "Did you mean to auto-size these brackets with \\left and \\right?", "line": "    (\\left(\\frac{a}{b}\\right))", "num": 19, "marker": [4, 29]}], "error": false}
